_config.yml

```diff
@@ -1,91 +1,73 @@
-title: 	NEON Raster Data in R 
-description: 	A set of lessons .
-#Comment out url when working locally to resolve base urls correctly ok
-#use this when pushing to git for live version
+#------------------------------------------------------------
+# Values for this lesson.
+#------------------------------------------------------------
 
-url:      http://datacarpentry.org/NEON-R-Spatial-Raster
-baseurl:   http://datacarpentry.org/NEON-R-Spatial-Raster
+# Which carpentry is this ("swc", "dc", or "lc")?
```

Comment:
@jsta they are all the carpentries now but this is a question for @tracykteal and @jduckles ... technically we originally started these for data carpentry but given they are merged now...